firmware/application/apps/acars_app.hpp

```diff
@@ -32,85 +32,69 @@
 #include "acars_packet.hpp"
 
 class ACARSLogger {
-public:
+ public:
```

Comment:
I hate this and I like this. It looks terrible, but it makes folding in my editor work better so...

Reply:
If you can find the clang parameter to stop it, let me know and I can add it in

Reply:
IndentAccessModifiers = false might do it.